### PR TITLE
Adds bonzofenix as approver for autoscaler team

### PR DESCRIPTION
**Pull request by bonzofenix**
List of previous contributions, see [RFC-0008](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md#revoking-approver-role):

https://github.com/cloudfoundry/app-autoscaler-release/commits?author=bonzofenix

### CODE REVIEW

**Comment by bonzofenix** [comment]
Hi @Gerg, could you please merge this PR according to the process described RFC-0008? Thanks!

**Review by silvestre — APPROVED** [review]

**Comment by stephanme** [comment]
Your contributions fulfil the approver requirements.
According to [rfc-0008-role-change-process.md](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md#promotion-to-approver), "two existing Approvers for that Working Group must attest that they meet the criteria by reviewing the PR."

I need one more +1 by an approver of your WG area, then I can merge.

File: org/cloudfoundry.yml [hunk 1]
```diff
@@ -2483,7 +2483,7 @@ orgs:
           cli-plugin-repo: admin
           cli: admin
           bosh-package-cf-cli-release: admin
-          cli-private: admin
```
Comment (geigerj0) [inline org/cloudfoundry.yml:2486]: 👍

**Review by geigerj0 — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Comment by stephanme** [comment]
Welcome @bonzofenix to the ARI WG.

**Pull request merged**